features/ubuntu_pro_fips.feature

```diff
@@ -0,0 +1,105 @@
+Feature: Command behaviour when auto-attached in an ubuntu PRO fips image
+
+    @series.lts
+    @uses.config.machine_type.azure.pro.fips
+    Scenario Outline: Check fips is enabled correctly on Ubuntu pro fips Azure machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I create the file `/etc/ubuntu-advantage/uaclient.conf` with the following:
+        """
+        contract_url: 'https://contracts.canonical.com'
+        data_dir: /var/lib/ubuntu-advantage
+        log_level: debug
+        log_file: /var/log/ubuntu-advantage.log
```

Comment:
Let's add this line here:
```
features:
  allow_xenial_fips_on_cloud: true
  ```
  
  To allow testing it on xenial for now

Reply:
Okay, really important.